src/Sylius/Component/Taxonomy/spec/Model/TaxonSpec.php

```diff
@@ -139,6 +139,19 @@ function its_fullname_prepends_with_parents_fullname(TaxonInterface $root): void
         $this->getFullname()->shouldReturn('Category / T-shirts');
         $this->getFullname(' -- ')->shouldReturn('Category -- T-shirts');
     }
+    
+    function its_fullname_prepends_with_parents_fullname_with_custom_separator(TaxonInterface $root): void
+    {
+        $root->getFullname()->withArguments([' / '])->willReturn('Category');
+        $root->getFullname()->withArguments([' -- '])->willReturn('Category');
+        $this->setName('T-shirts');
+
+        $root->addChild($this)->shouldBeCalled();
+        $this->setParent($root);
+
+        $this->getFullname()->shouldReturn('Category / T-shirts');
```

Comment:
I guess you wanted that custom argument here too?

Reply:
I want to test the default value.

Reply:
But the default value is tested in the preceding test.

Reply:
I test another behavior from the previous test